xc7/tests/counter/CMakeLists.txt

```diff
@@ -1,6 +1,13 @@
+#add_fpga_target(
+#  NAME counter
+#  BOARD basys3
+#  INPUT_IO_FILE basys3.pcf
+#  SOURCES counter.v
+#  )
+
 add_fpga_target(
```

Comment:
Probably worth creating a wrapper function "add_fpga_targets" that thats BOARD/INPUT tuples and calls add_fgpa_target